src/main/resources/assets/js/components/PieComponent.jsx

```diff
@@ -0,0 +1,82 @@
+/** @jsx React.DOM */
+
+define([
+  "React",
+  "d3"
+], function(React, d3) {
+  "use strict";
+
+  return React.createClass({displayName: 'Pie',
```

Comment:
PieComponent shouldn't be needed, but maybe a AppHealthBarComponent (depends on complexity of logic)?

Also, we tend to use this format:
return React.createClass({
    displayName: "AppHealthBarlComponent",